dask_ml/model_selection/_incremental.py

```diff
@@ -326,6 +356,14 @@ def fit(
         If RandomState instance, random_state is the random number generator;
         If None, the random number generator is the RandomState instance used
         by `np.random`.
+    verbose : bool, int, float, default=False
```

Comment:
Can you make all of these consistent? Assuming they have the same meaning everywhere.

I think the next line is incorrect, or at least unclear. The default is False so people won't see anything by default unless they've configured logging externally.